src/pages/workspace/withPolicy.tsx

```diff
@@ -97,11 +91,11 @@ const policyDefaultProps: WithPolicyOnyxProps = {
 /*
  * HOC for connecting a policy in Onyx corresponding to the policyID in route params
  */
-export default function <TProps extends WithPolicyProps, TRef>(WrappedComponent: ComponentType<TProps & RefAttributes<TRef>>): React.ComponentType<Omit<TProps, keyof WithPolicyOnyxProps>> {
+export default function <TRef, TProps extends WithPolicyProps>(WrappedComponent: ComponentType<TProps & RefAttributes<TRef>>): React.ComponentType<Omit<TProps, keyof WithPolicyOnyxProps>> {
```

Comment:
Why was TRef moved?

Reply:
@blazejkustra I was trying something that required me to move it but then I reverted it back, so I can move it back!